packages/core/src/types/AppConfig.ts

```diff
@@ -68,6 +69,9 @@ export interface AppOptions {
    */
   readonly moduleOverrides?: readonly ModuleConfigTuple[];
 
+  // TODO: ideally we can extract this from the osdkClient but it hides everything and provides no util.
```

Comment:
Should we file an issue on osdk-ts to track this?

Reply:
https://github.com/palantir/osdk-ts/issues/2288